### PR TITLE
Fix holorays staying if you switch holopads

### PR DESCRIPTION
**Pull request by ghost**
:cl: More Robust Than You
fix: Holorays are now properly deleted if you switch holopads automatically
/:cl:

### CODE REVIEW

File: code/game/machinery/hologram.dm [hunk 1]
```diff
@@ -311,7 +311,7 @@ Possible to do for anyone motivated enough:
 							continue
 						else
 							var/obj/machinery/holopad/pad_close = get_closest_atom(/obj/machinery/holopad, holopads, AI.eyeobj)
-							if(get_dist(pad_close, AI.eyeobj) <= holo_range)
+							if(get_dist(pad_close, AI.eyeobj) < holo_range)
```
Comment (Cyberboss) [inline code/game/machinery/hologram.dm]: Why?

Comment (MoreRobustThanYou) [reply code/game/machinery/hologram.dm]: Didn't really make sense

Comment (Cyberboss) [reply code/game/machinery/hologram.dm]: How does it not make sense?

Comment (MoreRobustThanYou) [reply code/game/machinery/hologram.dm]: nvm

**Pull request merged**